lib/Conversion/TorchToTMTensor/TorchToTMTensor.cpp

```diff
@@ -360,6 +360,222 @@ class ConvertAtenBincountOp : public OpConversionPattern<AtenBincountOp> {
 };
 } // namespace
 
+static SmallVector<SmallVector<int64_t>>
```

Comment:
Nit: can you add a small documentation string to this function? Without the context of the Python script, it's not very obvious what this function does